mapswipe_workers/mapswipe_workers/utils/api_calls.py

```diff
@@ -27,33 +29,76 @@ def geojsonToFeatureCollection(geojson: dict) -> dict:
     return geojson
 
 
-def query_osm(changeset_id: int):
+osm_results = {}
+
+
+def chunks(_list, n_objects):
+    return [
+        _list[i * n_objects : (i + 1) * n_objects]
+        for i in range((len(_list) + n_objects - 1) // n_objects)
+    ]
+
+
+def query_osm(changeset_ids: list):
     """Get data from changesetId."""
-    url = OSM_API_LINK + f"changeset/{changeset_id}.json"
+    id_string = ""
+    for id in changeset_ids:
+        id_string += f"{id},"
+
+    id_string = id_string[:-1]
 
+    url = OSM_API_LINK + f"changesets?changesets={id_string}"
+    logger.info(url)
     response = retry_get(url)
 
     if response.status_code != 200:
         err = f"osm request failed: {response.status_code}"
         logger.warning(f"{err}")
         logger.warning(response.json())
         raise CustomError(err)
-    response = response.json()["elements"][0]
-    return response
+    tree = ElementTree.fromstring(response.content)
+
+    for changeset in tree.iter("changeset"):
+        id = changeset.attrib["id"]
+        username = changeset.attrib["user"]
+        userid = changeset.attrib["uid"]
+        hashtags = created_by = None
+        for tag in changeset.iter("tag"):
+            if tag.attrib["k"] == "hashtags":
+                hashtags = tag.attrib["v"]
+            if tag.attrib["k"] == "created_by":
+                created_by = tag.attrib["v"]
+
+        osm_results[int(id)] = {
+            "username": username,
+            "userid": userid,
+            "hashtags": hashtags,
```

Comment:
sorry for the back end forth on this one here. Finally I come to the conclusion that we should use `comment` instead of `hashtags`. 

I've realized that the `hashtags` are only set if the user used the iD editor. If you use JOSM this is not set. However, both editors put the hashtags also in the comment.